projects/graph/src/queue.py

```diff
@@ -0,0 +1,26 @@
+
+class Queue:
+    def __init__(self):
+        self.queue = []
+    
+    def enqueue(self, value):
+        self.queue.append(value)
+    
+    def dequeue(self):
+        if self.size() > 0:
+            return self.queue[0]
+        else:
+            return None
+    
+    def is_Empty(self):
+        if self.size() == 0:
+            return False
+        else:
+            return True
+    
+    def size(self):
+        return len(self.queue)
```

Comment:
Good job putting the queue in another class file and importing it. That's the smart way to keep your file sizes down.